src/client/components/filter-menu/time-filter-menu/fixed-time-tab.tsx

```diff
@@ -402,23 +365,13 @@ export class FixedTimeTab extends React.Component<FixedTimeTabProps, FixedTimeTa
         </div>
       </div>;
     } else {
-      if (numTimeRanges < 5) {
-        return <div><div className="datePickers">{this.createDateRangePickers()}</div>
-          <div className="ok-cancel-bar">
-            <Button type="primary" onClick={this.onOkClick} disabled={!this.validate()} title={STRINGS.ok}/>
-            <Button type="secondary" onClick={onClose} title={STRINGS.cancel}/>
-            <Button type="secondary" title="Add Time Range" onClick={this.onTimeRangeClick}/>
-          </div>
-        </div>;
-      } else {
-        return <div><div className="datePickers">{this.createDateRangePickers()}</div>
-          <div className="ok-cancel-bar">
-            <Button type="primary" onClick={this.onOkClick} disabled={!this.validate()} title={STRINGS.ok}/>
-            <Button type="secondary" onClick={onClose} title={STRINGS.cancel}/>
-            <Button type="secondary" title="Add Time Range" onClick={this.onTimeRangeClick}/>
-          </div>
-        </div>;
-      }
+      return <div><div className="datePickers">{this.createDateRangePickers()}</div>
```

Comment:
Don't we also want `TimeShiftSelector` there? Maybe it's too much ... I'm not sure.

And `React.Fragment` is great tool for grouping nodes for React that doesn't appear in DOM.